Roys/app/src/main/java/com/dwg_karrier/roys/ContentView.java

```diff
@@ -47,6 +47,8 @@ protected void onCreate(Bundle savedInstanceState) {
     b.setOnClickListener(new View.OnClickListener() {
       @Override
       public void onClick(View v) {
+        ListActivity LA = (ListActivity) ListActivity.activity;
```

Comment:
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names
We already know this is a ListActivity variable. Can you give more specific name than LA to express what does this variable work for?